tests/todo.spec.js

```diff
@@ -0,0 +1,139 @@
+import {test, expect} from '@playwright/test'
+import { playwrightPage } from '../pom/playwrightPage';
+
+test.beforeEach(async ({page}) => {
+  let testpage = new playwrightPage(page);
+  await testpage.accessPage();
+});
+
+test.describe('reachability', () => {
+  test('check if URL is correct', async ({page}) => {
+    await expect(page).toHaveURL(/playwright/);
+  })
+  
+  test('check if title is correct', async ({page}) => {
+    await expect(page).toHaveTitle(/Todo/);
+  })
+  
+  test('check if footer buttons redirect to correct page and have correct text', async({page}) => {
+    let testpage = new playwrightPage(page);
+  
+    await expect(testpage.createdByLink).toHaveText('Remo H. Jansen');
+    await testpage.createdByLink.click();
+    await expect(page).toHaveURL('https://github.com/remojansen/');
+  
+    await testpage.accessPage();
+  
+    await expect(testpage.partOfLink).toHaveText('TodoMVC');
+    await testpage.partOfLink.click();
+    await expect(page).toHaveURL('https://todomvc.com/');
```

Comment:
considering that you are using the url within multiple tests, please add it to a separate _const_ variable